### DISCUSSION TITLE
Shut out?

### DISCUSSION THREAD

**Discussion by holmconsulting-morten** [teslamate-org/teslamate#2435] [General]
Teslamate is not able to authenticate and i'm not able to generate new API tokens. 

It might to be related to my IP address.

If i try to access http://tesla.com through my wifi-hotspot i get this:

<img width="527" alt="Screenshot 2022-02-01 at 22 04 47" src="https://user-images.githubusercontent.com/55490621/152051334-30ef051b-938c-4ff3-8d64-794d0a3c3f02.png">

And if i switch to using my mobile phone as hotspot i get the regular website.

Has Tesla shut me out? Does anyone have the same problems?

**Comment by cwanja** [comment]
Are you using a VPN? I believe Tesla blocks various VPN providers.
